addons/refuel/XEH_postInit.sqf

```diff
@@ -1,9 +1,77 @@
 #include "script_component.hpp"
+if (!GVAR(enabled)) exitWith {};
+
+["All", "InitPost", {
+    params ["_vehicle"];
+    if (getFuelCargo _vehicle <= 0) exitWith {};
+
+    if (local _vehicle) then {
+        _vehicle setFuelCargo 0;
+    };
+    if (-1 == _vehicle getVariable [QGVAR(HDEHID), -1]) then {
```

Comment:
Please stop doing that.

```suggestion
    if (_vehicle getVariable [QGVAR(HDEHID), -1] == -1) then {
```
and anywhere else.

Reply:
I would understand it for lines like `_fuel == 1` end maybe even `getFuelCargo _x < 1`. For such short conditions I usually use common conditions. But for long expression yoda condition looks much easier for me. I'm sorry but no.